dags/sync_log.py

```diff
@@ -0,0 +1,33 @@
+from airflow import DAG
+from datetime import datetime, timedelta
+from operators.emr_spark_operator import EMRSparkOperator
+from airflow.operators import BashOperator
+
+default_args = {
+    'owner': 'mreid@mozilla.com',
+    'depends_on_past': False,
+    'start_date': datetime(2017, 1, 1),
+    'email': ['telemetry-alerts@mozilla.com', 'mreid@mozilla.com'],
+    'email_on_failure': True,
+    'email_on_retry': True,
+    'retries': 2,
+    'retry_delay': timedelta(minutes=30),
+}
+
+dag = DAG('sync_log', default_args=default_args, schedule_interval='@daily')
+
+# Make sure all the data for the given day has arrived before running.
+t0 = BashOperator(task_id="delayed_start",
+                  bash_command="sleep 3600",
+                  dag=dag)
```

Comment:
No reason, other than that all the other DAGs use `@daily` too. I'll fix this one and `main_summary` which uses a similarly inefficient method to wait.